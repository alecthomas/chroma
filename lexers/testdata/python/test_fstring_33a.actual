f'*{n=!r:30}*'
