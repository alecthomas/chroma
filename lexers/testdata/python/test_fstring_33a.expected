[
  {"type":"LiteralStringAffix","value":"f"},
  {"type":"LiteralStringSingle","value":"'*"},
  {"type":"LiteralStringInterpol","value":"{"},
  {"type":"Name","value":"n"},
  {"type":"LiteralStringInterpol","value":"=!r:"},
  {"type":"LiteralStringSingle","value":"30"},
  {"type":"LiteralStringInterpol","value":"}"},
  {"type":"LiteralStringSingle","value":"*'"},
  {"type":"Text","value":"\n"}
]
